WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java

```diff
@@ -194,6 +210,13 @@ public GapMarkerViewHolder(View itemView) {
         }
     }
 
+    /*
+     * post header isn't shown for blog preview
+     */
+    private boolean shouldShowPostHeader() {
```

Comment:
Hey @nbradbury , this looks pretty much as the inverse of `isBlogPreview()`. May I suggest dropping is in favour of `isBlogPreview()`? It seems that there are quite a few comments around that will show what happens with the header when it is a blog preview.